nixos/modules/services/monitoring/prometheus/exporters/kafka.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.prometheus.exporters.kafka;
+  inherit (lib)
+    mkIf
+    mkOption
+    mkMerge
+    types
+    concatStringsSep
+    ;
+in {
+  port = 8080;
+
+  extraOpts = {
+    package = lib.mkPackageOption pkgs "kminion" { };
+
+    environmentFile = mkOption {
+      type = with types; nullOr path;
+      default = null;
+      description = ''
+        File containing the credentials to access the repository, in the
+        format of an EnvironmentFile as described by systemd.exec(5)
+      '';
+    };
+  };
+  serviceOpts = mkMerge ([{
+    serviceConfig = {
+      ExecStart = ''
+        ${cfg.package}/bin/kminion
+      '';
+      EnvironmentFile = mkIf (cfg.environmentFile != null) cfg.environmentFile;
+      RestartSec = "3s";
```

Comment:
I think we should keep the default 5s or is there a strong reason?

Reply:
The default actually seems to be the systemd one of `100ms` when the tests are ran.